data/retryable-reads/test_files.txt

```diff
@@ -15,8 +15,8 @@ distinct.json
 distinct-serverErrors.json
 estimatedDocumentCount-4.9.json
 estimatedDocumentCount-serverErrors-4.9.json
-estimatedDocumentCount-pre4.9.json
-estimatedDocumentCount-serverErrors-pre4.9.json
+estimatedDocumentCount.json
+estimatedDocumentCount-serverErrors.json
 find.json
```

Comment:
The Atlas Data Lake test file has also changed:
https://github.com/mongodb/specifications/commit/c43037606247a689cb5ada0099b8628a2c495bdd#diff-8170be3680c3fc86d77f1c4d3397c6c55c4438bcb6ae1b9a36965272c241044f

In the C++ driver, that file is in ./data/mongohouse/estimatedDocumentCount.json. `mongohouse` is the name of the process used for Atlas Data Lake. The change does not look significant, but I suggest updating it too since it is in the scope of the requested changes.